org.mixedrealitytoolkit.input/Utilities/FollowJoint.cs

```diff
@@ -32,46 +32,8 @@ internal class FollowJoint : MonoBehaviour, ISerializationCallbackReceiver
         [SerializeField, HideInInspector]
         private bool migratedSuccessfully = false;
```

Comment:
I think you can remove this now too

Reply:
Correct, did it already in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/774/commits/39287ee594ca665e3cca42376ff0802d3f9d5064 , resolving.

Reply:
@ms-RistoRK This doesn't look removed in that commit or this PR:

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/3580640/e558d36d-2224-45fc-a648-e08991bbc5ab)

Reply:
My bad.  Not it IS removed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/774/commits/73d9c6d25913e289ff08a9ecf274e1126c209fe4 , resolving.